Buffered Dump IO

StanleyMines opened the pull request:
This buffers the Input/Output streams for dumps, allowing dump formats that take many small operations to be much much faster. May help on existing dumps by just a little.

(I thought this was added a while ago.)

leMaik commented:
What's the difference between `FastBufferedInputStream` and `BufferedInputStream`? Is it worth not using the built-in one here?

StanleyMines commented:
> What's the difference between `FastBufferedInputStream` and `BufferedInputStream`? Is it worth not using the built-in one here?

Not too sure. The Fast one seems to have some other features, but also caused a crash when I added the others as Fast, so I left the existing one as Fast, and changed the new ones to just using the default Java one.

leMaik commented:
Can you make both ones the Java ones just to make sure we don't introduce regressions with this? :pray:

leMaik commented:
Als always, thank you! :+1:
